salt/_modules/caasp_pillar.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+
+def __virtual__():
+    return "caasp_pillar"
+
+
+def get(name, default=''):
+    '''
+    A sanitized version of pillar.get() that
+
+    * will return `default` (or `''`) instead of `None` or `''`
+    * will return `True`/`False` instead of `"true"`/`"false"`
+
+    The rationale is that, if pillar[x] can return None,
+    we can get a nasty "None" when replacing {{pillar[x]}} in a
+    jinja template. In other words, `None`s are annoying when used
+    in Salt/Jinja...
+    '''
+    res = __salt__['pillar.get'](name, default)
+    if not res:
+        res = default
```

Comment:
This will do a strange round trip if we have `False` (boolean type) as a pillar value for `name`, since `caasp_pillar.get` will return whatever the `default` value is instead of `False`.

Reply:
Good catch. I will use another solution.